gio/Gir.toml

```diff
@@ -997,6 +1002,34 @@ status = "generate"
     name = "set_child_setup"
     cfg_condition = "unix"
 
+[[object]]
+name = "Gio.Task"
+status = "generate"
+    [[object.function]]
+    name = "return_error"
+    #wrong mutability
+    ignore = true
+    [[object.function]]
+    name = "get_priority"
+    #wrong priority type
+    ignore = true
+    [[object.function]]
+    name = "set_priority"
+    #wrong priority type
+    ignore = true
+    [[object.function]]
+    name = "propagate_boolean"
+    #wrong return value
+    ignore = true
+    [[object.function]]
+    name = "return_value"
+    #wrong mutability
+    ignore = true
+    [[object.function]]
+    name = "propagate_value"
+    #wrong mutability
+    ignore = true
```

Comment:
`propagate_value()` and `return_value` seem rather important for proper usage from bindings so just implement them manually please :)

Reply:
I added the `value` ones and removed the `propagate_int()` and `return_int` and the like. I did that because looking at [task.c](https://github.com/frida/glib/blob/master/gio/gtask.c#L589), the result is implemented as a `union`, and nothing would prevent calling `return` with one type and after `propagate` with another type (for example `return_int` and `propagate_object`), and this would in general cause undefined behavior. Sticking to `Values` would probably cause some overhead but I think is safer. I'm open to suggestions of course :)

Reply:
Makes sense